docs/development.md

```diff
@@ -28,44 +30,120 @@ Plug-ins listed under the `plugins` array are then loaded and interrogated for k
+export MONGO_URL="mongodb://$MONGO_INITDB_ROOT_USERNAME:$MONGO_INITDB_ROOT_PASSWORD@localhost:27018"
+
 ```
 
-The provided configuration file allows some options to be assigned using environment variables.
+> [!TIP]
+> If you want to inspect data stored in a MongoDB database, you can use the [MongoDB shell](https://www.mongodb.com/products/tools/shell) or a GUI like [Compass](https://www.mongodb.com/products/tools/compass).
+
+## Configure the content store
+
+Indiekit performs CRUD operations on files. These files needs to be stored in a so-called content store. Different content stores require different configurations and credentials.
+
+### GitHub content store
+
+If you want to use a GitHub repository as your content store, you will need to provide a [GitHub personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens). If you create a [fine-grained personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#fine-grained-personal-access-tokens), give it **Read and write** access to **Contents**, and set a reasonable expiration (e.g. 90 days).
+
+```txt
+export GITHUB_USER="johndoe"
+export GITHUB_REPO="indiekit-content"
+export GITHUB_BRANCH="main"
+export GITHUB_TOKEN="github_pat_*****"
+```
+
+## Configure the syndicators
+
+In order to tell Indiekit to [share content with other third-party websites](introduction#sharing-content-with-third-party-websites-syndication), you need to configure one or more syndicators.
 
-Create an `.env` file in the root of the project, for example:
+### Mastodon syndicator
+
+If you want to syndicate your content to Mastodon, you will need to provide a Mastodon access token with **read and write** access. You can generate an access token using [this web app](https://takahashim.github.io/mastodon-access-token/), or making a POST request as explained [on this page of the Mastodon documentation](https://docs.joinmastodon.org/client/token/).
+
+> [!WARNING]
+> As far as I know, [Mastodon access tokens do not expire](https://mastodon.social/@kevinhooke/109377838604407902).
+
+```txt
+export MASTODON_ACCESS_TOKEN="*****"
+export MASTODON_USER="johndoe"
+export MASTODON_URL="https://fosstodon.org"
+```
+
+## Running Indiekit locally
+
+### Installation
+
+Install all dependencies:
 
 ```sh
-# Required
-PUBLICATION_URL="https://example.com"
-
-# Database connection string URI (optional)
-MONGO_URL="mongodb://127.0.0.1:27017"
-
-# Test saving files to a content store on GitHub (optional)
-GITHUB_USER="username"
-GITHUB_REPO="indiekit-test"
-GITHUB_BRANCH="main"
-GITHUB_TOKEN="12345abcde"
-
-# Test syndicating content to a Mastodon server (optional)
-MASTODON_URL="https://example.social"
-MASTODON_USER="indiekit-test"
-MASTODON_ACCESS_TOKEN="12345abcde"
+npm install
 ```
 
-You can then start the server:
+### Environment variables
```

Comment:
As mentioned, maybe this guidance should appear earlier in this document? Perhaps using `PUBLICATION_URL` as an example variable to set?

Happy to mention direnv, but the default means of setting variables should be using dotenv (perhaps to later be replaced with guidance that refers to Node’s native support for reading `.env` files).